projects/packages/forms/tests/php/contact-form/test-class.contact-form.php

```diff
@@ -965,7 +965,7 @@ public function getFirstElement( $dom, $tag, $index = 0 ) {
 	 * @param DOMElement $wrapper_div The wrapper div.
 	 * @param array      $attributes An associative array containing the field's attributes.
 	 */
-	public function assertCommonValidHtml( $wrapper_div, $attributes ) {
```

Comment:
This function used to check both classes and the label. I moved the label check out of it (for checkboxes and radio buttons, the field "label" is a `legend`) and renamed the function.